package.json

```diff
@@ -39,8 +38,7 @@
     "mongo:start": "sh ./scripts/start-mongo.sh '/var/lib/mongodb/'",
     "mongo:start:windows": "sh ./scripts/start-mongo.sh 'C:\\data\\db'",
     "data:local": "npm run clean:data && sh ./scripts/get-data-local.sh",
-    "cypress": "cypress",
-    "cypress:run": "cypress run --config-file ./config/cypress.config.js"
+    "e2e": "playwright test"
```

Comment:
Should the `e2e` script start the server, like the previous `test:e2e` script? Alternately, the [`webServer`](https://playwright.dev/docs/test-webserver) in the `playwright.config` could be updated to launch the server, if it isn't already running.

Reply:
Great idea! One observation was that I found instances where changing the playwright config would result in different results when rerunning tests and this lead me to question whether I need to restart the "app" and "server" instances for the new config to take.  For now, I think I'll just mimic the previous cypress behavior but I think I will revisit this once I dial in the playwright config.  Let me know if you had a similar experience.

Reply:
Oh interesting! I haven't run into that issue before, but makes sense to keep it as is for now